packages/lodestar-beacon-state-transition/src/epoch/finalUpdates.ts

```diff
@@ -16,7 +17,7 @@ export function processFinalUpdates(config: IBeaconConfig, state: BeaconState):
   const nextEpoch = currentEpoch + 1;
   // Reset eth1 data votes
   if (nextEpoch % config.params.EPOCHS_PER_ETH1_VOTING_PERIOD === 0) {
-    state.eth1DataVotes = [];
+    state.eth1DataVotes = [] as Eth1Data[] as List<Eth1Data>;
```

Comment:
I think these can be casted directly to `List<X>`

Reply:
Typescript errors with:
```
Conversion of type 'never[]' to type 'List<Eth1Data>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  The types returned by 'pop()' are incompatible between these types.
    Type 'undefined' is not comparable to type 'Eth1Data'.ts(2352)
```
Because it understands `[]` as `never[]`